src/Utility/HttpResponsePaginator.php

```diff
@@ -150,10 +150,13 @@ public function count(): int
     /**
      * Return the current result at our position, if available.
      *
-     * @return array<mixed>|bool
+     * @return mixed
+     *
+     * @psalm-suppress InvalidAttribute
      *
      * @codeCoverageIgnore
      */
+    #[\ReturnTypeWillChange]
```

Comment:
With PHP 8.1, an (arguably breaking) change was [implemented based on this RFC](https://wiki.php.net/rfc/internal_method_return_types) which sets "tentative" return types for internal methods. This requires code that overrides those methods to have a matching type. If the type does not match, it will throw a deprecation until PHP 9.